tsconfig.json

```diff
@@ -1,30 +1,21 @@
 {
   "compilerOptions": {
-    "typeRoots": [
-      "node_modules/@types",
-      "src/types"
-    ],
-    "target": "es5",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "allowJs": true,
+    "target": "ESNext",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": false,
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "noFallthroughCasesInSwitch": true,
+    "jsx": "react-jsx",
+    "types": ["vite/client", "vite-plugin-svgr/client"]
```

Comment:
using Vite runs, but I seem to be getting an issue on this line that states: ``` Cannot find type definition file for 'vite-plugin-svgr/client'. The file is in the program because: Entry point of type library 'vite-plugin-svgr/client' specified in compilerOptions ``` 

This link may have some useful information: https://github.com/microsoft/TypeScript/issues/54629

Reply:
Did you run `npm install` before running the dev server?